ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,173 @@
+base_images:
```

Comment:
Should we be in the OWNERS file for this directory? @ascerra @btjd

Reply:
I believe that we should be. This will allow for us to merge updates to this config if we need to. We may realize that we've made a mistake somewhere or implement a new tool like security checking that we'll want to be able to update this file without requesting PQE to do it.

Reply:
+1 it's good to have that option in case we need it.